bag/src/main/java/com/trempel/bag/ui/BagItemViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.trempel.bag.ui
+
+import android.view.View
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.navigation.findNavController
+import com.trempel.bag.model.BagDomainModel
+import com.trempel.core_ui.RecyclerItemComparator
+
+class BagItemViewModel(
+    val item: BagDomainModel
+) : ViewModel(), RecyclerItemComparator {
+
+    private val _quantity = MutableLiveData(item.quantity)
+    val quantity: LiveData<Int> get() = _quantity
+    val deleteLiveData = MutableLiveData<Int>()
+
+    fun onItemClicked(view: View, id: Int) {
+        val action = BagFragmentDirections.actionBagFragmentToNavGraphPdp(id)
+        view.findNavController().navigate(action)
+    }
+
+    fun deleteItem() {
+        deleteLiveData.value = item.id
+    }
+
+    fun increaseQuantityItem() {
+        _quantity.value = _quantity.value?.plus(1)
+    }
+
+    fun reduceQuantityItem() {
+        if (quantity.value ?: 0 > 1)
+            _quantity.value = _quantity.value?.minus(1)
```

Comment:
if statement should be one-lined or its body should be wrapped with curly braces